src/mame/act/victor9k.cpp

```diff
@@ -662,6 +683,7 @@ void victor9k_state::machine_reset()
 	m_via1->reset();
 	m_via2->reset();
 	m_via3->reset();
+	m_screen->reset();
```

Comment:
This should be removed.

Reply:
Thanks for the clarification. I removed it.